source/_posts/2015-04-28-open-sourcing-your-apps.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: How we Open Source'd Eigen 
+date: 2015-04-28
+comments: true
+categories: [ios, mobile, eigen]
+author: Orta Therox
+github-url: https://www.github.com/orta
+twitter-url: http://twitter.com/orta
+blog-url: http://orta.io
+---
+
+It was 95 days ago, and I was sitting before of my computer in NYC. I loaded my terminal, opened [TapTalk](https://taptalk.me) on my phone, tapped on my collegue [Ash's](http://ashfurrow.com) face and held my finger there. I paused, this moment was a long time, it was worth waiting. I switched the camera from my face to the screen with the terminal open.
+
+This moment was probably the most nervous I had been in years. It showed `git push origin master`. I said "it's happening." I hit return. A new era in the Artsy mobile team had started. A few minutes later, I wrote this tweet:
+
+<center>
+<blockquote class="twitter-tweet" data-cards="hidden" lang="en"><p>We have open source’d the <a href="https://twitter.com/artsy">@Artsy</a> iOS app.&#10;&#10;<a href="https://t.co/c1SWtHmUgy">https://t.co/c1SWtHmUgy</a>&#10;&#10;🎉</p>&mdash; Ørta (@orta) <a href="https://twitter.com/orta/status/558395611754819586">January 22, 2015</a></blockquote>
+</center>
+
+
+Let's go over the process we went through to get to that point.
+
+<!-- more -->
+
+Credit where credit is due, when we were [working on Eidolon](/blog/2014/11/13/eidolon-retrospective/), our CTO [dB](http://code.dblock.org/) just casually tossed the idea that, really, Eigen should be open source too. This totally threw me for a loop, we were only just getting to a point where we could build an app from scratch in the open. Trying to get a project that had existed for years and had it's own momentum converted would take a lot of thinking about.
```

Comment:
The first time you mention Eigen you should say that it's the mobile app, code name eigen.